scripts/analyze-houjin-quality.py

```
@@ -0,0 +1,234 @@
+  data/result/houjin_quality/{YEAR}/03_multi_name_per_cn.csv     1法人番号に複数名（表記揺れ）
+  data/result/houjin_quality/{YEAR}/04_name_diff_with_houjin.csv RS名とhoujin公式名の食い違い
+  data/result/houjin_quality/{YEAR}/06_cn_supplement_candidates.csv 番号欠落だが名称一致で補完候補
+
+05_type_code_mismatch は再生成しない:
+  現行 data/houjin.db の type_code は2桁コードで、NTA法人種別（3桁: 101/201/301…）を
+  信頼的に表現していない（例: 国土交通省・大阪府・岐阜県がいずれも '01'）。
+  アーカイブ版05が用いた3桁NTAコードは現DBから復元できないため、忠実再生成は不可。
+  法人種別の突合を行う場合は3桁NTAコードを持つ法人番号DBの再構築が前提（Phase 3の宿題）。
+
+法人番号の有効判定は app/lib/recipient-key.ts の isValidCorporateNumber と同一方針
+（13桁数字・全桁同一のダミーを除外）。チェックディジット検証は Phase 3 で追加予定のため本スクリプトでは行わない。
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)